rorio/README.md

```diff
@@ -0,0 +1,16 @@
+# ROR in OWL
+
+This [W3ID](https://w3id.org/) provides a persistent URI namespace for [Research Organization Registry (ROR)](https://ror.org) records in OWL resources.
+
+## Uses
+
+ROR in OWL is an ontology that wraps ROR identifiers as named individuals so they can be more easily included in other ontologies (e.g., OBO Ontologies) and empower the [Protégé](https://protege.stanford.edu/) ontology editor to show rich label information for `dc:contributor` annotations.
+
+The artifacts referenced by this entry are versionioned in the following GitHub repository: https://github.com/cthoyt/rorio
+
+## Contact
+
+Charles Tapley Hoyt
+Email: cthoyt@gmail.com
+GitHub: [@cthoyt](https://github.com/cthoyt/)
+ORCID: [0000-0003-4423-4370](https://orcid.org/0000-0003-4423-4370)
```

Comment:
Note typo: `versionioned` -> `versioned`

Also note line wraps are not preserved when MD is rendered.  Need to end each wrapping line with either two spaces (`  `) or an HTML BReak (`<br>`)